Snip/Globals.cs

```diff
@@ -34,6 +34,9 @@ public static class Globals
         public const string NewLineVariable = "$$n";
         public const string TrackIdVariable = "$$i";
 
+        // Delay when using WM_Char in ms
```

Comment:
It is strange I think that the window does not accept any input when holding ctrl and alt that is why I had to add a delay. I'm not a winapi export but there might be a way to signal that ctrl and alt are no longer pressed.